genesyscloud/resource_genesyscloud_outbound_settings_test.go

```diff
@@ -0,0 +1,201 @@
+					resource.TestCheckResourceAttr("genesyscloud_outbound_settings."+resourceId,
+						"automatic_time_zone_mapping.0.supported_countries.0", "CA"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_settings."+resourceId,
+						"automatic_time_zone_mapping.0.callable_windows.0.mapped.0.earliest_callable_time", "09:00"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_settings."+resourceId,
+						"automatic_time_zone_mapping.0.callable_windows.0.unmapped.0.latest_callable_time", "19:00"),
+				),
+			},
+			{
+				// Update all values
+				Config: generateOutboundSettingsResource(
+					resourceId,
+					"10",
+					"0.5",
+					"6.5",
+					complianceAbandonRateDenominator[0],
+					generateAutomaticTimeZoneMapping(
+						[]string{"US"},
+						generateCallableWindowsBlock(
+							generateMapped(
+								"08:00",
+								"18:00",
+							),
+							generateUnmapped(
+								"09:45",
+								"20:30",
+								"CET",
+							),
+						),
+					),
+				),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("genesyscloud_outbound_settings."+resourceId, "max_calls_per_agent", "10"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_settings."+resourceId, "max_line_utilization", "0.5"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_settings."+resourceId, "abandon_seconds", "6.5"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_settings."+resourceId, "compliance_abandon_rate_denominator", complianceAbandonRateDenominator[0]),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_settings."+resourceId,
+						"automatic_time_zone_mapping.0.supported_countries.0", "US"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_settings."+resourceId,
+						"automatic_time_zone_mapping.0.callable_windows.0.mapped.0.earliest_callable_time", "08:00"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_settings."+resourceId,
+						"automatic_time_zone_mapping.0.callable_windows.0.mapped.0.latest_callable_time", "18:00"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_settings."+resourceId,
+						"automatic_time_zone_mapping.0.callable_windows.0.unmapped.0.earliest_callable_time", "09:45"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_settings."+resourceId,
+						"automatic_time_zone_mapping.0.callable_windows.0.unmapped.0.latest_callable_time", "20:30"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_settings."+resourceId,
+						"automatic_time_zone_mapping.0.callable_windows.0.unmapped.0.time_zone_id", "CET"),
+				),
+			},
+			{
+				// Import/Read
+				ResourceName:            "genesyscloud_outbound_settings." + resourceId,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"max_calls_per_agent", "max_line_utilization", "abandon_seconds", "compliance_abandon_rate_denominator", "automatic_time_zone_mapping"},
+			},
+		},
+	})
```

Comment:
Dont you need a test to make sure the resource is deleted?

Reply:
Outbound settings only have two endpoints a get and a patch so I don't think I can delete the resource but I could be wrong?